health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/AppFieldScreenWrapper.js

```
@@ -0,0 +1,215 @@
+        }}
+      />
+      <Stepper
+        customSteps={[...stepper?.map((i) => i.name)]}
+        currentStep={currentStep}
+        onStepClick={() => {}}
+        activeSteps={0}
+        className={"appConfig-flow-stepper"}
+      />
+      {currentCard?.cards?.map(({ fields, description, header, headerFields }, index, card) => {
+        return (
+          <Card className="appConfigScreenCard">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)